### PR TITLE
add keys to config that's auto-loaded

### PR DESCRIPTION
**Pull request by mixmix**
In every client app I've ever touched this property `keys` is added manually to the config after `ssb-config` has done all its setup. 

I don't understand why this isn't part of the standard stuff loaded as part of `ssb-config` ... so here it is!

### CODE REVIEW

File: inject.js [hunk 1]
```diff
@@ -35,25 +36,26 @@ module.exports = function (name, override) {
     },
     connections: {
       outgoing: {
-        net: [{ transform: "shs" }]
+        net: [{ transform: 'shs' }]
       }
     },
-    path: path.join(HOME, '.' + name),
+    path: path,
+    keys: ssbKeys.loadOrCreateSync(join(path, 'secret')),
```
Comment (mixmix) [inline inject.js]: This is the only new line

**Comment by dominictarr** [comment]
sounds very reasonable, but did you actually confirm that this works with all those things?
do ssb-server tests still pass, can you run patch* etc

**Comment by mixmix** [comment]
I'll run those tests now then merge + publish if it's working fine

**Comment by dominictarr** [comment]
:+1:

**Comment by dominictarr** [comment]
@mixmix did you test this?

**Comment by mixmix** [comment]
I was waiting on the other stuff before piling more in. Now you've made a major bump and we got the testing working I'll go ahead and get this merged asap

**Comment by mixmix** [comment]
:heavy_check_mark: `ssb-server` tests pass

merging and publishing as 3.1.0

**Pull request merged**